libretro.c

```diff
@@ -312,6 +320,9 @@ void retro_set_environment(retro_environment_t cb)
 #ifdef HAVE_DYNAREC
       { "gpsp_drc", "Dynamic recompiler (restart); enabled|disabled" },
 #endif
+      { "gpsp_frameskip_type", "Frameskip type; off|manual" },
+      { "gpsp_frameskip_value", "Frameskip value; 1|2|3|4|5|6|7|8|9|0" },
```

Comment:
whats the difference between type=off and value=0?

Reply:
Great question; I copied most of the logic from the original PSP emulator, but that does seem a bit redundant especially without the auto frameskip option. Hopefully I'll get a chance to add that later.

To answer your question, there's more or less no difference. I guess turning it off would bypass some of the frameskip logic altogether, although I doubt it would really have a performance difference. Do you think it would be enough to remove `0` from frameskip value? Or would it be better to remove frameksip type altogether?